rosbag2_transport/package.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>rosbag2_transport</name>
+  <version>0.0.0</version>
+  <description>transport related dependencies for rosbag2</description>
```

Comment:
this description is not very helpful as it duplicates the name of the package... as i understand it this

"Extra layer that allows rosbag to be used with or without ROS2 middleware"

Reply:
I agree the description is not really precise. I'll touch up on it.

Reply:
Changed.